src/constants.js

```diff
@@ -952,6 +953,12 @@ export const styles = {
     width: 500
   },
 
+  modelSaveResponseDataMessage: {
+    display: "inline-block",
+    opacity: 0.5,
+    padding: "0 5px"
+  },
+
```

Comment:
Maybe a naive question: Why are css styles in a constants.js file?

Reply:
I don't think any particular reason, except apparently some early ones got added there and then it just stuck.  We could fix, though I'd prefer to isolate that to its own change.